### DISCUSSION TITLE
I want to change my commit username.

### DISCUSSION THREAD

**Discussion by jonah0704** [community/community#62787] [Discussions]
### Select Topic Area

Question

### Body

I'm trying to push my local changes to other's repository.
But commited username is different from my github username.
How to fix it?
Could anyone help me?

**Comment by dawn0123** [comment]
1. Open your terminal or command prompt.
2. Navigate to the root directory of your Git repository (if you want to change the configuration for a specific repository) or any directory (if you want to change the global configuration for all repositories).
3. Run the following command to change the user.name configuration:
git config [--global] user.name "Your New Name"
Then it works.

**Reply by jonah0704** [reply]
It really works. Thank you very much.

**Comment by jankozik** [comment]
thanks. it is helpfull.

**Comment by queenofcorgis** [comment]
@jonah0704 @risto1024 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
